site/content/docs/5.3/components/placeholders.md

```diff
@@ -80,7 +80,8 @@ Create placeholders with the `.placeholder` class and a grid column class (e.g.,
 
 We apply additional styling to `.btn`s via `::before` to ensure the `height` is respected. You may extend this pattern for other situations as needed, or add a `&nbsp;` within the element to reflect the height when actual text is rendered in its place.
 
-{{< example >}}
+<!-- Boosted mod: exclude-from-pa11y-analysis because the `a` here is seen as an anchor link -->
```

Comment:
Should it be a `Boosted mod` or a simple comment if you do it inside an Orange component?